src/Fixer/Alias/NoMixedEchoPrintFixer.php

```diff
@@ -42,14 +47,14 @@
     public function configure(array $configuration = null)
     {
         if (null === $configuration) {
-            $configuration = self::$defaultConfig;
-        } else {
-            if (1 !== count($configuration) || !array_key_exists('use', $configuration) || !in_array($configuration['use'], array('print', 'echo'), true)) {
-                throw new InvalidFixerConfigurationException($this->getName(), sprintf(
-                    'Expected array of element "use" with value "echo" or "print", got "%s".',
-                    var_export($configuration, true)
-                ));
-            }
+            $configuration = self::$defaultConfiguration;
+        }
+
+        if (1 !== count($configuration) || !array_key_exists('use', $configuration) || !in_array($configuration['use'], array('print', 'echo'), true)) {
```

Comment:
default configuration must not require extra validation.
one needs to validate user input, default configuration is hardcoded and tested by utests